dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/services/tasks/Ds3PutJob.java

```diff
@@ -89,6 +88,7 @@ public Ds3PutJob(final Ds3Client client,
             @Assisted final List<Pair<String, Path>> files,
             @Assisted("bucket") final String bucket,
             @Assisted("targetDir") final String targetDir,
+            @Assisted @Nullable final Ds3ClientHelpers.Job job,
```

Comment:
Instead of allowing for nullability from the caller, provide overloads that do not require it.